code/game/machinery/doors/airlock.dm

```diff
@@ -1249,11 +1249,9 @@ About the new airlock wires panel:
 			I.playtoolsound(loc, 100)
 			user.visible_message("[user] removes the electronics from the airlock assembly.", "You start to remove electronics from the airlock assembly.")
 			// TODO: refactor the called proc
-			if (do_after(user, src, 20))
-				to_chat(user, "<span class='notice'>You removed the airlock electronics!</span>")
-				revert(user,null)
-				qdel(src)
-				return
+			to_chat(user, "<span class='notice'>You removed the airlock electronics!</span>")
+			revert(user,null)
+			qdel(src)
```

Comment:
no more early return?

Reply:
Missed that, fixed. Adding fingerprints when installing electronics probably isn't wanted.